.github/workflows/app-e2e.yml

```diff
@@ -123,14 +123,14 @@ jobs:
           name: allure-results
           path: packages/app/allure-results
 
-      - name: Upload test results to Allure reporter
-        if: always()
-        env:
-          ALLURE_TOKEN: ${{ secrets.ALLURE_TOKEN }}
-        run: |
-          ./allurectl upload allure-results
-          echo "*Public report link: https://raw.githack.com/matter-labs/block-explorer/gh-pages/${{ github.run_number }}/index.html"
-          echo "*Public report link will be available when the 'Allure Report' job will be succesfully executed."
+      # - name: Upload test results to Allure reporter
```

Comment:
disable this GH action step for the time being as it fails for some reason. It will be investigated and fixed in the following PRs.